apps/editing-toolkit/editing-toolkit-plugin/global-styles/class-global-styles.php

```diff
@@ -543,4 +566,69 @@ public function filter_and_validate_font_options( $incoming_data ) {
 	}
 }
 
+function global_styles_fonts_message_control() {
+	if ( class_exists( 'Global_Styles_Fonts_Message_Control' ) ) {
+		return;
+	}
+
+	class Global_Styles_Fonts_Message_Control extends \WP_Customize_Control {
```

Comment:
What are the benefits of defining a class in a function?

Reply:
Basically I was adding conditional logic to prevent the class definition from being created more than once, if the file was included more than once.

I don't have a reason we can't just use `if ( ! class_exists )` without defining a function - I can switch over to that before this is complete.